substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/heap/Target_java_lang_ref_Reference.java

```diff
@@ -138,6 +139,12 @@ private void clear0() {
         clear();
     }
 
+    @Substitute
+    @TargetElement(onlyWith = JDK17_0_2OrLater.class)
+    boolean refersToImpl(T obj) {
+        return ReferenceInternals.refersTo(SubstrateUtil.cast(this, Reference.class), obj);
```

Comment:
Why does this method need an implementation? Can't it just be `@KeepOriginal` instead of `@Substitute` like `refersTo` itself?

Reply:
You are right, it's not needed. Fixed. Thanks!